test/Round-test.js

```diff
@@ -15,17 +15,19 @@ describe('Round', () => {
       { id: 2, name: 'Peerat' },
       { id: 3, name: 'Victor' }
     ]);
-    puzzle = new Puzzle(data.puzzles);
-    puzzle.choosePuzzleBank();
-    puzzle = puzzle.choosePuzzle();
+    // puzzle = new Puzzle(data.puzzles);
+    // puzzle.choosePuzzleBank();
+    // puzzle = puzzle.setPuzzle();
     round = new Round(puzzle);
+    round.startRound();
```

Comment:
Similar to my earlier comment about your puzzle class -- it's ok to immediately invoke some methods directly from your constructor to get all of this setup stuff out of the way. If you instantiate a new round, and the round's constructor invokes `startRound`, then you don't have to worry about doing it here.